framework/doc/content/syntax/Executioner/FixedPointAlgorithms/index.md

```diff
@@ -0,0 +1,61 @@
+# Fixed point iteration algorithms
+
+MOOSE provides fixed point algorithms in all its executioners. This can be used to iterate a single
+application solve to converge a parameter, for example converge the mass flow rate of a fluid simulation with a target pressure drop.
```

Comment:
I suggest to add some words in here to clarify the difference between this solve object and `FixedPoint` object like: it is noted that the lagged values can be stored in auxiliary variables, postprocessor values and/or user objects that are updated on `TIMESTEP_BEGIN` or `TIMESTEP_END` but we do not attempt to lag the solution vector for primal variables in this algorithm.

Reply:
I actually want these two solve objects to be merged. But this requires the vector adding all the time, which could be a waste if no lagged solution variables are used. So I am ok if we do not merge the two. `FixedPoint` is quite simple and provides a very useful feature actually.

Reply:
FixedPoint is a test object. I dont think we should refer to it in the documentation, that could get confusing.

I agree it's useful, but then it should be moved to the framework in a separate PR

Reply:
I was not asking for adding `FixedPoint` in that sentence. Please add the sentence because I think it is important to clarify this.

Reply:
ok added a note lower.